source/advanced_components/ddc/reference/provision/templates.rst

```diff
@@ -38,3 +38,78 @@ The following virtual network(s) are configured:
 
 * nat
 * private
+
+.. _ddc_provisison_cluster_templates:
+
+==========================
+Complete Cluster Templates
+==========================
+
+Several complete cluster templates are also shipped in the distribution package. These are for the two providers OpenNebula supports, Packet and AWS.
+
+.. _ddc_provision_template_packet:
+
+Packet Cluster Template
+-----------------------
+
+.. note::
+
+    Installed into
+    ``/usr/share/one/oneprovision/examples/example_packet.yaml``.
+
+Template with the following resources:
+
+- A CentOS7 host to be deployed in Packet (there is another with Ubuntu 18.04 commented)
+- Two SSH datastores, the system and the images ones.
+- A private network.
+- A public network to be deployed using the :ref:`Packet IPAM <ddc_ipam_packet>` and the :ref:`alias sdnat network driver <ddc_vnet_alias_sdnat>`.
+
+.. _ddc_provision_template_ec2:
+
+EC2 Cluster Template
+--------------------
+
+.. note::
+
+    Installed into
+    ``/usr/share/one/oneprovision/examples/example_ec2.yaml``.
+
+Template with the following resources:
+
+- A CentOS7 host to be deployed in AWS (there is another with Ubuntu 16.04 commented)
+- Two SSH datastores, the system and the images ones.
+- A private network.
+- A private network with nating.
+
+The way of using these example templates is :ref:`extending <ddc_usage_example6>` them with your own defaults sections. For example, imagine you want to deploy the Packet one:
+
+- First of all you need to have the template with the information needed by Packet and some other defaults:
+
+.. code::
+
+    $ cat base.yaml
+    ---
+    extends: example_packet.yaml
+    playbook: default
+    name: packet_provision
+
+    # Global defaults:
+    defaults:
+    provision:
+        driver: packet
+        packet_token: '***********************************'
+        packet_project: '*********************************'
+        facility: ams1
+        plan: baremetal_0
+    connection:
+        public_key: '/var/lib/one/.ssh/id_rsa.pub'
+        private_key: '/var/lib/one/.ssh/id_rsa'
+    configuration:
+        opennebula_node_kvm_param_nested: false
+
+    cluster:
+      name: packet-cluster
+
+.. note:: You need to put a valid token and project.
+
```

Comment:
Let's instruct user only to
- copy example template
- replace `*****` with credentials
- uncomment hosts

==> for that, example template must be really nearly complete (with some defaults and playbook).

We'll wait with "extends" way for provision inputs.